src/redux/selectors/cellSets/getCellSets.js

```diff
@@ -2,6 +2,20 @@ import createMemoizedSelector from 'redux/selectors/createMemoizedSelector';
 
 import initialState from '../../reducers/cellSets/initialState';
 
-const getCellSets = () => (state) => (Object.keys(state).length ? state : initialState);
+const getCellSets = () => (state) => {
+  const stateToReturn = (Object.keys(state).length ? state : initialState);
+
+  const accessible = !(
+    stateToReturn.loading
+    || stateToReturn.initialLoadPending
+    || stateToReturn.updatingClustering
+    || stateToReturn.error
+  );
+
+  return {
+    ...stateToReturn,
+    accessible,
+  };
```

Comment:
Why not just store `accessible` as a property? I think the idea of having `accessible` is good. However, returning it as an additional export from `getCellSets()` forces us to always use `getCellSets()` if we want to check for accesibility.

Reply:
I think it's better to store no more data than the absolute necessary (which makes it easy to keep the data consistent and avoid weird behaviors because we forget to set some property that depends on other props) and then derive properties from that data stored.

However, I do see your point on the fact that having it in `getCellSets()` makes us dependent on it. To deal with that I could add another selector that returns whether cell sets is `accessible` or not to make this accessible property more directly usable. (I saw some examples of this usage in the redux page)

Reply:
<img width="820" alt="image" src="https://user-images.githubusercontent.com/8465903/182614736-c750067c-edd6-4d16-bbb4-e797a8b0ed69.png">

> We specifically recommend that Redux apps should [keep the Redux state minimal, and derive additional values from that state whenever possible](https://redux.js.org/style-guide/#keep-state-minimal-and-derive-additional-values).

https://redux.js.org/usage/deriving-data-selectors

Reply:
Hmm... then instead of returning an object with 2 items, maybe return null if the cellSets is not ready? I don't think it's nice to have another selector just to check accessibility

Reply:
the new selecotr is memoized so there isn't any performance impact there)

Reply:
> maybe return null if the cellSets is not ready
if we return `null` then we can't read whether it was an error or what happened exactly

Reply:
Resolved over slack, we decided to keep the original version with `accessible` in `getCellSets`